Lib/test/test_typing.py

```diff
@@ -3134,6 +3134,24 @@ def bar(self, x: str) -> str:
 
         self.assertIsInstance(Test(), PSub)
 
+    def test_pep695_generic_protocol_callable_members(self):
+        @runtime_checkable
+        class Foo[T_co](Protocol):
```

Comment:
Pretty sure this is a T_contra :)

Reply:
changed my mind, now I hate Hungarian notation